### DISCUSSION TITLE
Using geometry triggers: Error querying the database: db error: ERROR: type "geometry" does not exist when applying following migration

### DISCUSSION THREAD

**Discussion by anooppoommen** [prisma/prisma#9381] [Q&A]
I've got a schema  like the following 

```prisma
model BlockObject {
  Id                Int                   @id @default(autoincrement())
  geo_point         Unsupported("geometry")?
}

```

when I create it everything seems to work out absolutely fine but when I go to run an other migration it complains with 
geometry not found. 
Migration `20210922091148_adding_geo_data` failed to apply cleanly to the shadow database. 
Error:
Database error
Error querying the database: db error: ERROR: type "geometry" does not exist
   0: sql_migration_connector::flavour::postgres::sql_schema_from_migration_history
             at migration-engine/connectors/sql-migration-connector/src/flavour/postgres.rs:375
   1: migration_core::api::DevDiagnostic
             at migration-engine/core/src/api.rs:89

**Comment by ryands17** [comment]
@anooppoommen 👋 

Try the following steps:

1. Create the migration file with `prisma migrate dev --create-only`
2. Edit the generated `.sql` file and add the line as shown [here](https://github.com/prisma/prisma-examples/blob/latest/typescript/postgis-express/prisma/migrations/20210819135741_/migration.sql)
3. Run `prisma migrate dev` to apply the migration

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
